### DISCUSSION TITLE
How to compile to WASM

### DISCUSSION THREAD

**Discussion by abdulbadii** [official-stockfish/Stockfish#5440] [Q&A] [answered]
How to compile to WASM ie. where is the link to help, or could it be pretty simple so it's proper to be shown here in concise yet clear?

**Comment by vondele** [comment]
The WASM port is not maintained here, but various forks exist. I'm not quite sure where the most recent versions are but

https://github.com/fairy-stockfish/fairy-stockfish.wasm 
https://github.com/lichess-org/stockfish.wasm
https://github.com/hi-ogawa/Stockfish

Are good places to start.

**Reply by niklasf** [reply, marked as answer]
The latest and probably best of this kind is https://github.com/lichess-org/lila-stockfish-web.
